.github/CODEOWNERS

```diff
@@ -0,0 +1,75 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+# Last matching entry wins.
+
+# Default owner for all missed or newly added files.
+*                                   @magicznyleszek @noliveleger
+
+
+
+
+#### Global section.
+.editorconfig                       @magicznyleszek
+.git-blame-ignore-revs              @magicznyleszek
+.gitattributes                      @magicznyleszek @noliveleger
+.gitignore                          @magicznyleszek @noliveleger
+.gitmodules                         @magicznyleszek @noliveleger
+CONTRIBUTING.md                     @magicznyleszek
+dependabot.yml                      @magicznyleszek
+kpi.code-workspace                  @magicznyleszek 
+LICENSE                             @magicznyleszek @noliveleger @jnm
+README.md                           @magicznyleszek @noliveleger
+
+
+
+
+#### Frontend section.
+
+# Default owner
+.storybook/*                        @magicznyleszek
+cypress/*                           @magicznyleszek
+jsapp/*                             @magicznyleszek
+patches/*                           @magicznyleszek
```

Comment:
@p2edwards Wold you like to take ownership of patches?

Reply:
done